### PR TITLE
[travis] Remove PHP 7 from allowed failures

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

Looking at the latests PRs, PHP 7 does not segfault anymore with our test suite.

### CODE REVIEW

**Comment by stof** [comment]
:+1:

**Comment by nicolas-grekas** [comment]
php 7 doesn't agree with us, still failing :)
Status: needs work

**Comment by nicolas-grekas** [comment]
We could work around this failure by moving to a higher lowest version of Doctrine. Should we do it or wait for PHP to fix the issue?

**Comment by nicolas-grekas** [comment]
After some investigation, the version bump is legitimate and related to a bc break in PHP7 (related to a new behavior of func_get_all()). Doctrine ORM 2.4.5 [fixes the issue](https://github.com/doctrine/doctrine2/pull/1142) and is required on PHP7.
Status: needs review

File: composer.json [hunk 1]
```diff
@@ -63,8 +63,8 @@
     "require-dev": {
         "symfony/phpunit-bridge": "~2.7",
         "doctrine/data-fixtures": "1.0.*",
-        "doctrine/dbal": "~2.2",
-        "doctrine/orm": "~2.2,>=2.2.3",
+        "doctrine/dbal": "~2.4",
+        "doctrine/orm": "~2.4,>=2.4.5",
```
Comment (stof) [inline composer.json]: use ``^2.4.5`` here

**Comment by stof** [comment]
the bump is not an issue IMO. Doctrine 2.2 and 2.3 are long EOLed

File: composer.json (hunk 1 above)
Comment (nicolas-grekas) [inline composer.json]: this would be the first use in the code base. I'd prefer not doing it in this PR

Comment (Tobion) [inline composer.json]: we use the caret in symfony standard as well by now. so I agree with stof.

Comment (nicolas-grekas) [inline composer.json]: also agree, but in an other PR :)

Comment (stof) [inline composer.json]: I forgot it was not yet used in Symfony itself

**Comment by Tobion** [comment]
:+1:

**Comment by Tobion** [comment]
Thank you @nicolas-grekas.

**Pull request merged**